### PR TITLE
do not run handle_post when depends_on is empty

### PR DESCRIPTION
**Pull request by lisps**
isset(array()) returns true

this fixes an issue, which accours if fieldset has no label

### CODE REVIEW

File: helper/fieldfieldset.php [hunk 1]
```diff
@@ -66,7 +66,7 @@ function renderfield($params, Doku_Form $form, $formid) {
      * @return bool Whether the passed value is valid
      */
     public function handle_post($value, &$fields, $index, $formid) {
-        if(!isset($this->depends_on)) {
+        if(!empty($this->depends_on)) {
```
Comment (splitbrain) [inline helper/fieldfieldset.php]: this seems wrong. It actually reverses the logic. Should it be a check on ``empty`` instead of ``!empty``?

Comment (lisps) [inline helper/fieldfieldset.php]: you are fast and right, thanks!

Comment (lisps) [inline helper/fieldfieldset.php]: unittest should have failed ;-)

**Comment by lisps** [comment]
fixed with #179

**Pull request closed without merging**